### ISSUE TITLE
Matias Ergo Pro - Unacceptable Key Switch Failure Rate

### ISSUE THREAD

**Issue by reyjrar** [diimdeep/awesome-split-keyboards#43]
Over the course of two years, I had key switches fail on *six* different Matias Ergo Pro's. No spills, no drops, no abuse, and only 40h/week use.  Some keys would stop registering input, other keys would bounce back up, but would repeat the character infinitely until unplugged and plugged back in.

It was the most comfortable keyboard I've ever used, but I would exclude it from the list because the key switches are terrible.  FWIW, last I checked, the beautiful Keyboard.io used the same key switches and has similarly high failure rates. Both are beautiful and comfortable, but *six* keyboards all failed in 2 years for just me.  Only 3 were replaced under warranty.  On the other hand, I'm typing this on a 12 yr old Apple chiclet keyboard that's never had issues, despite having been spilled on, dropped, and shipped internationally twice.

**Comment by benji-york** [comment]
As someone interested in keyboards, I really appreciate this input.

As a project outsider, I'm not sure how the information can be integrated.

Perhaps reported issues can be linked from the keyboard list.

**Comment by nitsujri** [comment]
I can also confirm on the Ergo Pro's failure rate. It's annoying and quite high. I left that keyboard behind years ago and don't recommend it to anyone.

I don't think we should integrate reviews into this repo though. Starts leading to gamification long term.

**Comment by reyjrar** [comment]
I read "Awesome Split Keyboards" as a review. The Matias Ergo Pro is awesome in theory. In practice, it's a terrible keyboard because of the reliability issues. I do not find an expensive keyboard I rely on to be "awesome" if it fails 6 times in two years.

I would just make a strong vote for "Remove Matias Ergo Pro" from the list.  Someone trusting this list might get into a situation where their $250 keyboard is worthless to them and out of warranty.

And to keep this thread up-to-date in case someone stumbles onto it, the Keyboard.io keyboards no longer use the key switches that fail constantly in the Matias Ergo Pro.  So, they can be declared "awesome"! :)

**Comment by swebb** [comment]
I agree with this. The keyboard is extremely comfortable but the switches keep failing. I've gone through 3 keyboards. My current solution is using of my replacements for spares (de-soldering and re-soldering the switches).

**Comment by wmaouad** [comment]
I second, and third, and fourth, and tenth the request to remove thee Matias Ergo Pro from the awesome keyboards list. It's not awesome. It's "potentially awesome". I had to use my backspace 2 dozen times already typing this, to remove duplicated characters.

Now I will typepe with no backspacing at all (except for my own typos): thee keey switches on this keeyboard are nice for a ffew months; Then it goes downhill very quickly. It's eexpensive, for a very risky product, a bad warranty, and no support at all. Futhermoree, it's design is very 1990's clunky, the "leegs" for rising iit up are not thought through, no stability at all unless flat on the surface (no using thee raisers at all).

For the price, regarding the performance andd durability of this keyboard (you would expect a keyb for life at that price point with Alps-inspired switches made by the company itself), it is a very, very, very bad iinvestment.

- Unusable (without hacking away at the switches yourself)
- Unsellable

Not awesome, please reemove from the list!
